src/PyHyperScattering/SST1RSoXSDB.py

```diff
@@ -331,13 +331,13 @@ def summarize_run(self, outputType:str = 'default', cycle:str = None, proposal:s
                             warnString =("Scan: > " + str(currentScanID) + " < Failed to locate metaData entry for > " 
                                          + str(outputVariableName) + " <\n Tried looking for label: > " 
                                          + str(metaDataLabel) + " < in: " + str(metaDataSource))
-                            warnings.warn(warnString,stacklevel=2)
+                            #warnings.warn(warnString,stacklevel=2)
                             
                     except (KeyError,TypeError):
                         warnString =("Scan: > " + str(currentScanID) + " < Failed to locate metaData entry for > " 
                                      + str(outputVariableName) + " <\n Tried looking for label: > " 
                                      + str(metaDataLabel) + " < in: " + str(metaDataSource))
-                        warnings.warn(warnString,stacklevel=2)
+                        #warnings.warn(warnString,stacklevel=2)
```

Comment:
Same thing, warning shouldn't be commented out.
```suggestion
                        warnings.warn(warnString,stacklevel=2)
```